languages/python/sqlalchemy-oso/sqlalchemy_oso/session.py

```diff
@@ -231,6 +232,11 @@ def __init__(self, oso: Oso, user, checked_permissions: Permissions, **options):
         responses from changing, ensuring that the session's identity map never
         contains unauthorized objects.
         """
+        if oso._polar_roles_enabled:
+            raise UnsupportedError(
+                "Data filtering not yet supported with Polar roles enabled."
```

Comment:
We should be directing people to using the sqlalchemy builtin roles

Reply:
good idea

Reply:
8fb1530